app/uk/gov/hmrc/individualincomedesstub/domain/Employment.scala

```diff
@@ -111,7 +111,7 @@ object EmploymentIncomeResponse {
         employment.startDate.map(parse), employment.endDate.map(parse), desPayFrequency, employment.payments map (DesPayment(_))
```

Comment:
Since the employer has been found using that empRef, they're one and the same. 
However, done.